cmd/sops/main.go

```diff
@@ -1387,7 +1388,11 @@ func main() {
 				},
 				cli.BoolFlag{
 					Name:  "value-file",
-					Usage: "treat 'value' as a file to read the actual value from (avoids leaking secrets in process listings)",
+					Usage: "treat 'value' as a file to read the actual value from (avoids leaking secrets in process listings). Mutually exclusive with --value-stdin",
+				},
+				cli.BoolFlag{
+					Name:  "value-stdin",
+					Usage: "treat 'value' as a file to read the actual value from (avoids leaking secrets in process listings). Mutually exclusive with --value-file",
```

Comment:
This help text is wrong: `--value-stdin` doesn't take a value, only `--value-file` does that.

Reply:
Neither option takes a value. They both modify what the `value` parameter of `sops set` means.